sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation? DeepChem and DragoNN are worth
 discussing here.*
 
-### Transfer learning/transferability of features
+### Multimodal and transfer learning
 
 * https://github.com/greenelab/deep-review/issues/139#issuecomment-268901804
+
+As discussed above, the fact that biomedical datasets often contain a limited
+number of instances or labels, often leads to poor performance of
+machine learning algorithms. When trained on such datasets, deep learning
+models are particularly prone to overfitting due to their high representational
+power. However, Transfer Learning techniques also known as Domain Adaptation
```

Comment:
I don't think Transfer Learning should be caps (it's not in the title). Domain Adaptation probably shouldn't be either.